src/why/Why.tsx

```diff
@@ -10,11 +11,27 @@ export interface WhyProps {
   readonly onClick?: () => void
 }
 
-const Why = ({ className, children, title, onClick }: WhyProps) => (
-  <button type="button" className={cc(['kirk-why', className])} title={title} onClick={onClick}>
-    <QuestionIcon />
-    <span>{children}</span>
-  </button>
-)
+const Why = ({ className, children, title, onClick }: WhyProps) => {
+  const { focusVisible, onFocus, onBlur } = useFocusVisible()
```

Comment:
@simonrelet to answer your question about the `useFocusVisible` implementation, here is this example with the `Why` component.